src/User.php

```diff
@@ -23,6 +23,8 @@ class User implements UserInterface
 	public $title;
 	public $forename;
 	public $surname;
+	public $description;
+	public $jobTitle;
```

Comment:
@kuiche I wonder if perhaps it would be an idea to have user 'types' in the same way we have product and page types. Obviously, we wouldn't want such a feature to be too intrusive (setting a default type with no fields), but it might resolve this issue and allow us to freely add more custom fields to users in the future. Just a thought